docs/book/provider_implementations/create_actuators.md

```diff
@@ -106,7 +106,8 @@ const (
 )
 
 // Add RBAC rules to access cluster-api resources
-//+kubebuilder:rbac:groups=cluster.k8s.io,resources=machines;machines/status;machinedeployments;machinedeployments/status;machinesets;machinesets/status;machineclasses,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=cluster.k8s.io,resources=machines;machines/status,verbs=get;list;watch;create;update;patch;delete
```

Comment:
The Machine Actuator should not need to interact with MachineSets or Machine Deployments, and the MachineClass interaction should not be as broad.